src/Service/JobOfferService.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Service;
+
+use App\Entity\Company;
+use App\Entity\JobOffer;
+use App\Request\PostJobOffer;
+use App\Request\CompanyRegistration;
+use App\Repository\JobOfferRepository;
+
+final class JobOfferService
```

Comment:
Should we add a test for this service?

Reply:
I don't think so. We already have it covered by integration test. I can't see any edge case that'd deserve a unit test.